assets/src/blocks/Timeline/TimelineEditor.js

```diff
@@ -0,0 +1,206 @@
+  {label: 'Japanese', value: 'ja'},
+  {label: 'Korean', value: 'ko'},
+  {label: 'Latvian', value: 'lv'},
+  {label: 'Lithuanian', value: 'lt'},
+  {label: 'Luxembourgish', value: 'lb'},
+  {label: 'Malay', value: 'ms'},
+  {label: 'Myanmar', value: 'my'},
+  {label: 'Nepali', value: 'ne'},
+  {label: 'Norwegian', value: 'no'},
+  {label: 'Polish', value: 'pl'},
+  {label: 'Portuguese', value: 'pt'},
+  {label: 'pt-br', value: 'Portuguese (Brazilian)'},
+  {label: 'Romanian', value: 'ro'},
+  {label: 'Romansh', value: 'rm'},
+  {label: 'Russian', value: 'ru'},
+  {label: 'Serbian - Cyrillic', value: 'sr-cy'},
+  {label: 'Serbian - Latin', value: 'sr'},
+  {label: 'Sinhalese', value: 'si'},
+  {label: 'Slovak', value: 'sk'},
+  {label: 'Slovenian', value: 'sl'},
+  {label: 'Spanish', value: 'es'},
+  {label: 'Swedish', value: 'sv'},
+  {label: 'Tagalog', value: 'tl'},
+  {label: 'Tamil', value: 'ta'},
+  {label: 'Taiwanese', value: 'zh-tw'},
+  {label: 'Telugu', value: 'te'},
+  {label: 'Thai', value: 'th'},
+  {label: 'Turkish', value: 'tr'},
+  {label: 'Ukrainian', value: 'uk'},
+  {label: 'Urdu', value: 'ur'},
+];
+
+const position = [
+	{label: 'Bottom', value: 'bottom'},
+	{label: 'Top', value: 'top'},
+]
+
+let url_desc  = __(
+	'Enter the URL of the Google Sheets spreadsheet containing your timeline data.',
+	'planet4-blocks-backend'
+);
+url_desc += '<br><a href="https://timeline.knightlab.com/#make" target="_blank" rel="noopener noreferrer">';
+url_desc += __(
+	'See the TimelineJS website for a template GSheet.',
+	'planet4-blocks-backend'
+);
+url_desc += '</a><br>';
+url_desc += __(
+	'Copy this, add your own timeline data, and publish to the web.',
+	'planet4-blocks-backend'
+);
+url_desc += '<br>';
+url_desc += __(
+	"Once you have done so, use the URL from your address bar (not the one provided in Google's 'publish to web' dialog).",
+	'planet4-blocks-backend'
+);
+
+const renderEdit = ({ google_sheets_url, language, timenav_position, start_at_end }, toAttribute) => {
+  // Using a state to prevent the input losing the cursor position, a React issue reported multiple times
+  const [ sheetURL, setSheetURL ] = useState(google_sheets_url);
+  const debounceSheetURLUpdate = useCallback(debounce(toAttribute('google_sheets_url'), 300), []);
+
+  return (
+    <InspectorControls>
+      <PanelBody title={__('Setting', 'planet4-blocks-backend')}>
+        <URLInput
+          label={__('Google Sheets URL', 'planet4-blocks-backend')}
+          placeholder={__('Enter URL', 'planet4-blocks-backend')}
+          help={ <RawHTML>{url_desc}</RawHTML> }
+          value={sheetURL}
+          onChange={
+            value => {
+              setSheetURL(value)
+              debounceSheetURLUpdate(value)
+            }
+          }
+      />
+
+        <SelectControl
+          label={__('Language', 'planet4-blocks-backend')}
+          value={language}
+          options={languages}
+          onChange={toAttribute('language')}
```

Comment:
Hmmm you're right, it was like this before WYSIWYG but I guess it doesn't make much sense. As this is feature change, I would split it to another ticket just in case we are missing something, like a post actually having a Timeline in a language that is not the post's language..., but indeed seems unlikely.